specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/2016-08-10/service.json

```diff
@@ -2303,6 +2343,69 @@
           }
         }
       },
+      "put": {
+        "tags": [
+          "ReplicationRecoveryServicesProviders"
+        ],
+        "summary": "Adds a recovery services provider.",
+        "description": "The operation to add a recovery services provider.",
+        "operationId": "ReplicationRecoveryServicesProviders_Create",
```

Comment:
@veronicagg For Job model this is the PR"https://github.com/Azure/azure-rest-api-specs/pull/1209" where i guess it was closed. Since we are not changing the version of the API we are not making that change.